src/main/java/de/triology/cas/ldap/ConnectionAwareAuthenticator.java

```diff
@@ -45,6 +47,17 @@ public ConnectionAwareAuthenticator(
         this.useUserConnectionToFetchAttributes = useUserConnectionToFetchAttributes;
     }
     
+    public void someBuggyMethod() {
+        LdapEntry entry = null;       
+        if (entry == null) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS2583)